client/src/pages/signUpPage.tsx

```diff
@@ -57,6 +58,15 @@ const SignUpPage: React.FC = () => {
             placeholder="enter your email"
             required
           />
+          <FormLabel>Mobile Number</FormLabel>
+          <TextField
+            id="email"
```

Comment:
@striker0234 the id of mobile number field is `email`.

Please take care of these things